packages/nimble-components/src/table-column/anchor/cell-view/template.ts

```diff
@@ -26,7 +26,7 @@ export const template = html<TableColumnAnchorCellView>`
                 target="${x => x.columnConfig?.target}"
                 type="${x => x.columnConfig?.type}"
                 download="${x => x.columnConfig?.download}"
-                underline-hidden="${x => x.columnConfig?.underlineHidden}"
+                ?underline-hidden="${x => x.columnConfig?.underlineHidden}"
```

Comment:
Looks like the matrix tests did the same thing, 

https://github.com/ni/nimble/blob/9f63cc8bb43b3c28fe1ccdec774e05693de1e885/packages/nimble-components/src/table-column/anchor/tests/table-column-anchor-matrix.stories.ts#L66

Could you update the binding and make the matrix test use true|false instead of true|undefined?